metashare/api/models.py

```diff
@@ -42,6 +42,17 @@ def refresh_repositories(self):
         GitHubRepository.objects.bulk_create(
             [GitHubRepository(user=self, url=repo) for repo in repos]
         )
+        self.notify_repositories_updated()
+
+    def notify_repositories_updated(self):
+        from .serializers import RepositorySerializer
+
+        gh_repositories = self.repositories.values_list("url", flat=True)
+        repositories = Repository.objects.filter(repo_url__in=gh_repositories)
```

Comment:
@wlonk It looks like this entirely bypasses pagination. It also seems a bit odd to me that we're duplicating similar logic here and in the `RepositoryViewSet`. I'd propose either returning no content here and allowing the front end to query the `RepositoryViewSet` endpoint, or making this `payload` actually use the `RepositoryViewSet` (including pagination).

Reply:
For now, we'll remove the payload from the `USER_REPOS_REFRESH` event and allow the front-end to trigger a new `GET`.